content/blog/2024-05-02-flash-expert.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "A User's Perspective on the Updated FlaSH System"
+author: Tina Townes
+date: 2024-05-02
+tags:
+  - flash
+authors:
+  - tina
+heroImage: blog-thumb-flash-expert.png
+heroImageThumb: blog-lg-flash-expert.png
+summary: | 
+  A reflection of the recent changes to the FlaSH user experience by our quality assurance expert, Tina Townes. 
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+<center>
+![**Fig 1a.** Revised FlaSH Dashoard](/blog/2024-05-02-flash-expert/new_dash.png)</center>
+
+
+In its initial stages, the FlaSH dashboard (Fig 1b) only enabled me to assess potential anomalies by viewing graphs, line-by-line, as generated by the FlaSH program. There wasn’t an efficient way to filter various incoming anomalies when I needed to examine specific geographic areas or signals. Nor was there an easy way to see a daily overview map of the aggregated average FlaSH scores for nationwide anomalies. Without the current dashboard, I was spending a good amount of time scrolling, manually sorting, documenting, and searching for specific anomaly reports I wanted to examine rather than focusing solely on identifying, marking, and analyzing anomalies.
```

Comment:
w.r.t. "graphs, line-by-line": might wanna expand on this and mention that there is a "line" for each location of each signal that has a flagged anomaly, and that there are a relatively large number of these to go through.  More example images or scenarios might help.

w.r.t. "manually sorting": what does this interaction look like and what makes it so undesirable?

Reply:
I've forwarded this link to Tina and will let her cover content questions!

Reply:
Hi, sorry for the delay! Thanks so much for the suggestions, here are my edits. Please let me know if more clarification is needed:

In its initial stages, the FlaSH dashboard (Fig 1b) only enabled me to assess potential anomalies by viewing graphs, line-by-line for each location of the numerous signals that have flagged anomalies, as generated by the FlaSH program. This was a particularly daunting task as daily FlaSH outputs generated and continue to produce a large number of reports in the form of compressed lines that required clicking on to expand and reveal more details. Without the new dashboard's features, I was spending a significant amount of time scrolling through the daily list of anomaly reports and manually sorting what I wanted to review by clicking on and expanding only certain report lines and leaving them expanded until I was done with my selection process and ready to review the expanded lines. I would also often make notes and document interesting patterns in anomalies in a separate notepad, decreasing the efficiency and speed of my review process. My attention became divided as I was parsing though the daily anomaly list to search for reports in certain geographies (I knew I wanted to examine these due to prior report patterns), while simultaneously trying to focus on assessing new anomalies.

With the old dashboard setup, it was not easy for me to review the lines of daily anomaly reports because I couldn't efficiently filter various incoming anomalies when I needed to examine specific geographic areas or signals. For example, one particular week I was seeing a lot of anomaly reports in a county in Puerto Rico Monday through Wednesday. By Thursday of that week I wanted to, upon logging into the platform, immediately proceed to filter the daily anomaly reports to look specifically at that Puerto Rican county right away, but had no way of filtering by geography with the old dashboard. The updated dashboard now has a side menu that lets me efficiently select to filter lines not only by the geographic regions, but also by various indicators as well. This new setup speeds up my daily review process as it lets me quickly focus on specific geographies and finish reviewing those so that I can move on and focus on examining other anomaly reports in different geographies.

Furthermore, with the former FlaSH dashboard setup, I didn't have an easy way to get a quick bigger picture of geography of the day's flagged anomalies. The old dashboard didn't provide an overview map of the aggregated average FlaSH scores for nationwide anomalies, I would log into the platform and just see a list of daily reported anomalies. Here, only after I assessed a significant number of reported anomalies would I develop a sense of the geographies where significant anomalies were occurring. The updated platform now immediately upon logging in displays a heat map that provides me with an overview of the locations and severities of the day's anomaly reports. This overview map is color-coded to highlight the geographies with the day's with highest aggregated FlaSH scores indicated in dark red on a sliding scale ending with the lowest scores in light red. This new map also speeds up my daily review process by giving me a heads-up on geographies I may want to focus on for the day before I even begin assessing the first anomaly report.